tests/v2/validation/rbac/rbac_etcd_backup_test.go

```diff
@@ -53,7 +53,11 @@ func (rb *ETCDRbacBackupTestSuite) SetupSuite() {
 func (rb *ETCDRbacBackupTestSuite) ValidateEtcdSnapshotCluster(role string) {
 
 	log.Infof("Creating a snapshot of the cluster as %v", role)
-	err := etcdsnapshot.CreateSnapshot(rb.standardUserClient, rb.clusterName, rb.namespace)
+
+	// clusterID, err := clusters.GetClusterIDByName(rb.client, rb.clusterName)
+	// require.NoError(rb.T(), err)
+	// clusterResponse, err := rb.client.Steve.SteveType("provisioning.cattle.io.cluster").ByID(clusterID)
```

Comment:
If we don't need this commented out code please remove

Reply:
removed